scp/cc/azure/attestation/src/attestation.h

```diff
@@ -0,0 +1,149 @@
+/*
+ * Portions Copyright (c) Microsoft Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ATTESTATION_H
+#define ATTESTATION_H
```

Comment:
Partially addressed in https://github.com/KenGordon/data-plane-shared-libraries/pull/11/commits/07e343f8e551dc0ad542bb58f5eb81c9de34613f

I've included the important parts of the path, but left out `src` since I think that's messier looking